cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -278,21 +278,14 @@ class ArchitectureOsValidator(Validator):
     ARM AMIs are only available for a subset of the supported OSes.
     """
 
-    def _validate(self, os: str, architecture: str, custom_ami: str, ami_search_filters):
```

Comment:
Is it because this check is only used for centos?

Reply:
Yes